src/pages/linux/how-to-set-up-ubuntu-lamp-server/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: how-to-set-up-ubuntu-lamp-server

```

Comment:
This needed to be changed @fvcproductions

Reply:
Ah, okay - I'll go ahead and make those changes.